.flake8

```
@@ -18,7 +18,9 @@ ignore =
     D301,
     D400, D401, D403, D404
     # ignored by pydocstyle numpy docstring convention
-    D107, D203, D212, D213, D402, D413, D415, D416, D417,
+    D107, D203, D212, D402, D413, D415, D416, D417,
+    # while D213 is also part of numpy docstring convention, it's left out above,
```

```suggestion
    # while D213 is also ignored by numpy docstring convention, it's left out above,
```
I may have misunderstood, but the [numpydoc styleguide](https://numpydoc.readthedocs.io/en/latest/format.html) doesn't seem to care about this rule.

---

Yes, thanks.